fdbrpc/fdbrpc.h

```diff
@@ -425,7 +425,10 @@ struct serializable_traits<RequestStream<T>> : std::true_type {
 		} else {
 			const auto& ep = stream.getEndpoint();
 			serializer(ar, ep);
-			UNSTOPPABLE_ASSERT(ep.getPrimaryAddress().isValid());  // No serializing PromiseStreams on a client with no public address
+			if constexpr (Archiver::isSerializing) { // Don't assert this when collecting vtable for flatbuffers
```

Comment:
```suggestion
			if constexpr (if_fb_function<Archiver>) { // Don't assert this when collecting vtable for flatbuffers
```
I think this would be cleaner

Reply:
As far as I can tell, the ASSERT would never be called if I did that (at least when serializing with flatbuffers)

Reply:
ah I see, I misunderstood what you were trying to do here…